src/main/java/rs/teslaris/core/model/user/RefreshToken.java

```diff
@@ -20,7 +19,6 @@
 @AllArgsConstructor
 @Entity
 @Table(name = "refresh_tokens")
-@Where(clause = "deleted=false")
```

Comment:
Why this should be removed? Probably not linked with upgrade of spring boot

Reply:
I don't know, now I see that it is deprecated. I have to research that.

Reply:
From integration testing and table inspections, switching to @SQLRestriction seems good. We will test a little bit manually but I think this is solved.

Reply:
Source: https://stackoverflow.com/questions/77178963/is-there-a-replacement-for-the-in-6-3-deprecated-where-and-loader